### DISCUSSION TITLE
Creeper Help?

### DISCUSSION THREAD

**Discussion by xipsaaahoy** [GriefPrevention/GriefPrevention#1245] [Q&A]
Does grief prevention take away creeper damaging the player?

**Comment by Jikoo** [comment]
GriefPrevention will block creeper explosions in claimed areas as long as the members of the claim have not toggled claim explosions on. Outside of claims, creeper damage is blocked above sea level if `BlockSurfaceExplosions` is true in your configuration.

**Comment by RoboMWM** [comment]
explosion damage to player entities is not protected, but any animals within a claim will be protected.
